src/components/dashboard/dashboard-list/DashboardList.tsx

```diff
@@ -38,21 +44,269 @@ class DashboardList extends Component<DashboardListProps, DashboardListState> {
+                            </div>
+                            <div className="level-item">
+                                <p>Interview Score </p>
+                                <input
+                                    className="slider has-output-tooltip is-large"
+                                    step="0.5"
+                                    min="0"
+                                    max="5"
+                                    value="3"
+                                    type="range"
+                                ></input>
+                                <output htmlFor="sliderWithValue">3</output>
+                            </div>
+                        </div>
+                        <div className="level-right">
+                            <div className="level-item">
+                                <button className="button is-light is-rounded is-link">Bulk Accept</button>
+                                <button className="button is-light is-rounded is-danger">Bulk Reject</button>
+                            </div>
+                        </div>
+                    </div>
+                );
+            case 'Final Decision':
+                return (
+                    <div className="level">
+                        <div className="level-left">
+                            <div className="level-item">
+                                <p>Application Score </p>
+                                <input
+                                    className="slider has-output-tooltip is-large"
+                                    step="0.5"
+                                    min="0"
+                                    max="5"
+                                    value="3"
+                                    type="range"
+                                ></input>
+                                <output htmlFor="sliderWithValue">3</output>
+                            </div>
+                            <div className="level-item">
+                                <p>Interview Score </p>
+                                <input
+                                    className="slider has-output-tooltip is-large"
+                                    step="0.5"
+                                    min="0"
+                                    max="5"
+                                    value="3"
+                                    type="range"
+                                ></input>
+                                <output htmlFor="sliderWithValue">3</output>
+                            </div>
+                        </div>
+                        <div className="level-right">
+                            <div className="level-item">
+                                <button className="button is-light is-rounded is-info">Bulk Email</button>
+                            </div>
+                        </div>
+                    </div>
+                );
+        }
+    };
```

Comment:
Just to shorten this file + maintainability I think it would be a good idea to change the contents of the conditional rendering here to their own components